Cubical/Categories/Instances/Functors.agda

```diff
@@ -70,6 +70,9 @@ module _ (C : Category ℓC ℓC') (D : Category ℓD ℓD') where
   Iso→NatIso α .trans = α .fst
   Iso→NatIso α .nIso = FUNCTORIso' _ (α .snd)
 
+  NatIso→Iso : {F G : Functor C D} → NatIso F G → CatIso FUNCTOR F G
```

Comment:
This is kind of a general comment, but wouldn't it be better if lemmas about `CatIso` would be called something with `CatIso` and not just `Iso`? So this would be `NatIso→CatIso`

Reply:
I change it to `FUNCTORIso` so it's more informative.

Reply:
(Look not very pretty though...)

Reply:
Yeah, the all caps names are not so nice... We should discuss naming convention for CT in an issue

Reply:
Ok. Then that's for it now.